### DISCUSSION TITLE
Edited beatmaps not treated as such

### DISCUSSION THREAD

**Discussion by plaguemask** [ppy/osu#15987] [Q&A] [answered]
It seems I've found some problems with how the game deals with beatmaps that I've edited locally. It's like the game treats them as if they hadn't been edited at all. (version 2021.1207.0)

 - Previous local records are not erased (and attempting to replay throws a `BeatmapNotFoundException`: [runtime.log](https://github.com/ppy/osu/files/7670766/runtime.log))

 - SR doesn't update, which seems to give the ranking panel incorrect pp.
Here's the result after replacing the contents of The Solace of Oblivion with some buzz sliders:
![osu!_2021-12-07_11-25-16](https://user-images.githubusercontent.com/66150801/145081431-d1eef1f0-9879-43cf-96d7-ef5f5855ba57.png)
The SR is still equal to the original map, but it should be less than 1. The pp should also be about 0.
(Restarting fixes this) 

 - Plays still submit and appear on profiles:
![firefox_2021-12-07_11-39-28](https://user-images.githubusercontent.com/66150801/145081707-3159b36e-73ab-4f2f-8ed6-b3d09f42d633.png)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/11922
https://github.com/ppy/osu/issues/12048

etc etc, although the score submitting is a larger issue. that needs server-side hash validation.
